src/panels/lovelace/hui-editor.ts

```diff
@@ -40,13 +50,26 @@ class LovelaceFullConfigEditor extends hassLocalizeLitMixin(LitElement) {
           <app-toolbar>
             <paper-icon-button
               icon="hass:close"
-              @click="${this.closeEditor}"
+              @click="${this._closeEditor}"
             ></paper-icon-button>
             <div main-title>Edit Config</div>
+            ${
+              this._hash
```

Comment:
Feels a bit weird to put in the title. If there is not enough space (ie on mobile), it will look very weird. It also won't work as an element above the text area because showing it will push the whole textarea down, and that's annoying. Wonder if this should be a toast that's shown when entering a `#` character?

Reply:
I was thinking about a toast too but figured it should be close to the save button...